### DISCUSSION TITLE
Not allowed to navigate to filesystem URL

### DISCUSSION THREAD

**Discussion by ronaldohoch** [chinchang/web-maker#519] [Q&A]
Hello, thank you for this beautiful app/extension!

Does one of you has seem these error: 
![image](https://user-images.githubusercontent.com/2287371/183078046-8bb46a64-0b73-425a-befe-4d66c7acbadd.png)

I'm using Brave Browser: `[Versão 1.42.86 Chromium: 104.0.5112.81 (Versão oficial) 64 bits](https://brave.com/latest/)`, yesterday was working... 😟

**Comment by chinchang** [comment]
Hi Ronaldo!
I just updated by Brave browser to 104 chromium and it stopped working for me too. Infact even Chrome 104 is behaving the same!
Seems like they have introduced some new policy which is causing this.

While I investigate this, here is what I would recommend:
If you are already logged in, then good. Otherwise, please login first of all and save all your creations in your account. Then go to https://webmaker.app/app/ and login again with your same account. You'll get all your creations on the web app.
Also, its recommended anyways to switch to the web app since it has more features and it works offline too.

I'll keep updating my findings here.

**Comment by chinchang** [comment]
This is fixed now.
